src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -2344,4 +2344,12 @@
     <value>This option is managed by enterprise policy and cannot be changed here.</value>
     <comment>This is displayed in concordance with Globals_StartOnUserLogin if the enterprise administrator has taken control of this setting.</comment>
   </data>
+  <data name="Appearance_BackgroundImageNone" xml:space="preserve">
+    <value>None</value>
+    <comment>Text displayed when the background image path is not defined.</comment>
+  </data>
+  <data name="Profile_AnswerbackMessageNone" xml:space="preserve">
+    <value>None</value>
+    <comment>Text displayed when the answerback message is not defined.</comment>
+  </data>

```

Comment:
Figured these needed to be localized in case the language needed more context. Like, I know in Spanish, we should use "ninguno" vs "ninguna" based on pronouns.